src/lib/setup.js

```diff
@@ -240,6 +240,14 @@ export default function(app) {
     })
   }
 
+  // Routes for pinging system time and up
```

Comment:
Would it be helpful to add a comment about the order of mounting and why this needs to be here?

Reply:
I was hoping that the commit message for https://github.com/artsy/force/pull/3203/commits/ba67f40462f8df69456b718462127f25acd51fa1 would be the right place to provide context. Here is it again for convenience: 

```
Load health check endpoints before others

Before this change: the profile app was redirecting
all traffic of that didn’t explicitly match another route to it’s
app. This was resulting in slow health checks that were integrating with
gravity. Considering these health check requests run every 5 seconds,
we’re putting a lot of needless traffic on gravity and other potential
downstream services.

* prettify src/lib/setup.js

Co-authored-by: Christopher Pappas <damassi.pappas@gmail.com>
``` 

Generally, I try to push context as much to commit messages as possible. Also, it seems inline with our [guidelines in artsy/README](https://github.com/artsy/README/blob/380b65bb0cffe3b8e1c45254eb3800b7016b2885/playbooks/engineer-workflow.md#commits).

Happy to move this to the source if that's the standard for this codebase though. LMK!

Reply:
A commit message should indeed provide context for why the change was necessary, but the comment I’m asking for isn’t really about the change, just an explanation of the importance of order when it comes to these things. The next person that will perform a refactor in this file will likely not look through the commit history for this file.